ui/guest/src/global.d.ts

```diff
@@ -14,23 +14,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React from 'react';
-
-declare global {
-  interface Window {
-    tinymce: any;
-    ace: AceAjax.Ace;
-  }
-  type CrafterCMSCustomElementProps = React.DetailedHTMLProps<
-    Omit<React.HTMLAttributes<HTMLDivElement>, 'className'> & { class?: string },
-    HTMLDivElement
-  >;
-  namespace JSX {
-    interface IntrinsicElements {
-      'craftercms-zone-marker': CrafterCMSCustomElementProps;
-      'craftercms-asset-uploader-mask-container': CrafterCMSCustomElementProps;
-      'craftercms-asset-uploader-mask': CrafterCMSCustomElementProps;
-      'craftercms-field-instance-switcher': CrafterCMSCustomElementProps;
-    }
-  }
+interface Window {
+  tinymce: any;
+  ace: AceAjax.Ace;
 }
```

Comment:
@rart I was getting a weird behavior when having `declare global {`. First time I did `build:modern` it was working, but after trying again with other build (like `build:node`) it started failing (with the issue mentioned on chat).
While reading, I found [this](https://stackoverflow.com/questions/57132428/augmentations-for-the-global-scope-can-only-be-directly-nested-in-external-modul) (see last comment) they mention that `declare global` is not needed on `.d.ts` files if they have no export. So I tried and it works properly in both react 18 and 19.